src/pwa/components/ShareBar/Shares.js

```diff
@@ -78,28 +78,44 @@ const Container = styled.div`
   box-sizing: border-box;
   width: calc(100vw - 130px);
   display: flex;
+  justify-content: space-around;
+  align-items: center;
   height: ${({ theme }) => theme.heights.bar};
   flex-grow: 1;
 
   & > div {
+    height: 42px;
+    width: 42px;
+    padding: 2px;
+    flex-grow: 0;
     color: ${({ theme }) => theme.colors.white};
-    flex-grow: 1;
     display: flex;
     align-items: center;
     justify-content: center;
-    width: ${({ theme }) => theme.heights.bar};
   }
 `;
 
 const Box = styled.div`
-  background-color: ${({ theme, color }) => theme.colors[color]};
-  max-width: 54px;
-
   & > * {
+    background-color: ${({ theme, color }) => theme.colors[color]};
     width: 100%;
     height: 100%;
     display: flex;
     justify-content: center;
     align-items: center;
+    border-radius: 50%;
   }
 `;
+
```

Comment:
I think I commented this because it was breaking the build when I did the merge with `mst-migration` but it can be removed.